scripts/allowlist/createContent.js

```diff
@@ -0,0 +1,101 @@
+const BN = require("bn.js");
+const fs = require("fs");
+const XRegExp = require('xregexp');
+const { sha3, soliditySha3, padLeft } = require("web3-utils");
+
+var text = fs.readFileSync("./allowedAccountsAndValidators.txt", "utf-8");
```

Comment:
It would be good style to not have any top level code  but call everything from some main function.

Reply:
Done